examples/hal/wifi/partitions.csv

```diff
@@ -1,5 +0,0 @@
-# ESP-IDF Partition Table
```

Comment:
this one will most probably be needed because final binary cannot fit into default 1M partition